examples_tests/01.HelloWorld/main.cpp

```diff
@@ -286,28 +286,22 @@ Choose Graphics API:
 		auto gpus = apiConnection->getPhysicalDevices();
 		assert(!gpus.empty());
 
-
-
 		// Find a suitable gpu
 		uint32_t graphicsFamilyIndex(~0u);
 		uint32_t presentFamilyIndex(~0u);
 
-		// Todo(achal): Probably want to put these into some struct
 		uint32_t minSwapchainImageCount(~0u);
 		video::ISurface::SFormat surfaceFormat;
 		video::ISurface::E_PRESENT_MODE presentMode;
 		asset::E_SHARING_MODE imageSharingMode;
 		VkExtent2D swapchainExtent;
 
-		// Todo(achal): Look at this:
-		// https://github.com/Devsh-Graphics-Programming/Nabla/blob/6bd5061abe0a2020142efda827269ea6c07f0f2f/examples_tests/common/CommonAPI.h
 		for (size_t i = 0ull; i < gpus.size(); ++i)
 		{
 			gpu = gpus.begin()[i];
 
 			bool isGPUSuitable = false;
 
-			// Todo(achal): Abstract out

```

Comment:
they're still your TODOs

Reply:
Oh I thought they irrelevant now, since its all in the CommonAPI anyway. But if you want I can add this comments back in and do the TODOs?

Reply:
then use a part of the CommonAPI which does all this

Reply:
I think this is no longer relevant.

Resolved?

Reply:
yep pawned off to @deprilula28 and also deciding to always use CommonAPI now